hollow-jar/web/pom.xml

```diff
@@ -27,11 +27,40 @@
             <artifactId>jboss-jaxrs-api_2.0_spec</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+           <groupId>javax.faces</groupId>
+           <artifactId>javax.faces-api</artifactId>
+           <scope>provided</scope>
+        </dependency>
+        <dependency>
+           <groupId>javax.validation</groupId>
+           <artifactId>validation-api</artifactId>
+           <scope>provided</scope>
+        </dependency>
```

Comment:
Please either use the `org.jboss.spec` APIs, or replace all the API dependencies here by this single uberjar: `org.jboss.spec:jboss-javaee-web-7.0:1.1.1.Final`